ext/drmgr/drmgr.h

```diff
@@ -216,6 +216,7 @@ typedef enum {
     DRMGR_PHASE_ANALYSIS,      /**< Currently in the analysis phase. */
     DRMGR_PHASE_INSERTION,     /**< Currently in the instrumentation insertion phase. */
     DRMGR_PHASE_INSTRU2INSTRU, /**< Currently in the instru2instru phase. */
+    DRMGR_PHASE_POST_INSTRU,   /**< Currently in the post instrumentation phase. */
```

Comment:
I'm rethinking whether we should name it "post instru". Post-instrumentation could refer to any phase after `DRMGR_PHASE_INSERTION`. Maybe a name that describes the phase itself, rather than its placement would be better.

Reply:
Maybe instru_debug or something?

Reply:
> Went with post-instru because in early review stages someone mentioned that it might be more than just for debugging, 

Can you remind me what additional uses we thought the additional phase can support? Maybe we can name "meta instru" or something.